src/app/ballot/page.tsx

```diff
@@ -233,8 +229,9 @@ function YourBallot() {
   const displayProjects = searchTerm ? filteredProjects : projectList;
 
   const isMovable =
-    distributionMethod !== DistributionMethod.IMPACT_GROUPS &&
-    distributionMethod !== DistributionMethod.CUSTOM;
+    distributionMethod === DistributionMethod.TOP_TO_BOTTOM ||
+    distributionMethod === DistributionMethod.TOP_WEIGHTED ||
+    distributionMethod === DistributionMethod.CUSTOM;
```

Comment:
This explains one of the issues I'm dealing with. This value was previously correct being:
```typescript=
const isMovable =
    distributionMethod !== DistributionMethod.IMPACT_GROUPS &&
    distributionMethod !== DistributionMethod.CUSTOM;
```

You should not be able to reorder the projects when `CUSTOM` distribution method is selected (issue #187).